de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/IESRImportBL.java

```diff
@@ -122,4 +121,10 @@
 	 * @param bankStatementLineRef
 	 */
 	void unlinkESRImportLinesFor(I_C_BankStatementLine_Ref bankStatementLineRef);
+
+	/**
+	 *
+	 * @param runESRImportRequest
+	 */
+	public void scheduleESRImportFor(final RunESRImportRequest runESRImportRequest);
```

Comment:
```suggestion
	public void scheduleESRImportFor(RunESRImportRequest runESRImportRequest);
```

* avoid useless javadocs
* final makes no sense in this interface method